### PR TITLE
grade-school: maintain via test_gen

### PR DESCRIPTION
**Pull request by marionebl**
* On top of #362
* Via #284

### CODE REVIEW

File: exercises/grade-school/test.ml [hunk 1]
```diff
@@ -7,24 +7,269 @@ let assert_lists_equal exp got _ctx =
...
+     assert_lists_equal [] (grade 1 s)
+     );
+     "an empty school has Emma, Timmy, Bob and Becky in grade 1 after they've been added to grade 1" >::(
+     let s = empty_school |> add "Emma" 1 |> add "Timmy" 1 |> add "Bob" 1 |> add "Becky" 1 in
+     assert_lists_equal ["Becky"; "Bob"; "Emma"; "Timmy"] (grade 1 s |> List.sort compare)
+     );
+     "a sorted school has Emma, Timmy, Bob and Becky in grade 1 in sorted order" >::(
+     let s = empty_school |> add "Emma" 1 |> add "Timmy" 1 |> add "Bob" 1 |> add "Becky" 1 |> sorted in
+     assert_lists_equal ["Becky"; "Bob"; "Emma"; "Timmy"] (grade 1 s)
+     ); *)
```
Comment (sshine) [inline exercises/grade-school/test.ml]: Curious, are these commented-out tests a byproduct of the test generator?

Comment (marionebl) [reply exercises/grade-school/test.ml]: Just clumsy me - forgot to remove the commented out old tests I left there for reference

**Comment by sshine** [comment]
@marionebl, @iHiD: Blocking merges until a reviewer approves was recently turned on for this repo. Do I remember wrong when saying that what I requested was to block merges if CI failed? In either case, I'd like for the "Review required" flag to be removed; sometimes @marionebl has PRs hanging for 7+ days, and I'd like for him to be able to merge without me if he thinks the subject is insignificant enough.

**Comment by iHiD** [comment]
@sshine I don't remember to be honest. Either way: Review is now off. CI enforcement is now on :+1:

**Pull request merged**